### ISSUE TITLE
Fail to install jruby-9.1

### ISSUE THREAD

**Issue by ms-ati** [travis-ci/travis-ci#9049]
It remains impossible to test using the `jruby-9.1` target.

Please see https://travis-ci.org/ms-ati/docile/jobs/327830706

```bash
$ rvm use jruby-9.1 --install --binary --fuzzy
curl: (22) The requested URL returned error: 404 Not Found
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Required jruby-9.1.15.0200 is not installed - installing.
curl: (22) The requested URL returned error: 404 Not Found
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Searching for binary rubies, this might take some time.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0200.
Requested binary installation but no rubies are available to download, consider skipping --binary flag.
Gemset '' does not exist, 'rvm jruby-9.1.15.0200 do rvm gemset create ' first, or append '--create'.
The command "rvm use jruby-9.1 --install --binary --fuzzy" failed and exited with 2 during .
```

This open issued at the request of @BanzaiMan to separate it from the separate problem in #8446 (missing jruby downloads archives), which was resolved previously.

**Comment by BanzaiMan** [comment]
The problem is twofold.

1. The file in question does exist on JRuby's S3 bucket. This is the error number 1. This should be fixed by the JRuby team. See https://github.com/jruby/jruby/issues/4967
1. RVM cannot handle this name, even though the archive is valid and continue to download this archive (whether or not this is an error in archive naming). This should be fixed by RVM.

In the mean time, you can use an explicit version name, such as `jruby-9.1.15.0` until either of the above issues is fixed.

**Comment by BanzaiMan** [comment]
The JRuby issue should be fixed now. `jruby` should now work. https://travis-ci.org/BanzaiMan/travis_production_test/builds/328243602.

**Comment by ms-ati** [comment]
@BanzaiMan Cool! Does it also fix `jruby-9.1`?

**Comment by sushilrai** [comment]
Observing failure with Travis Test

`$ mkdir -p ~/.gradle && echo "org.gradle.daemon=false" >> ~/.gradle/gradle.properties
11.68s$ rvm use jruby-9.1.15.0 --install --binary --fuzzy
curl: (22) The requested URL returned error: 404 Not Found
curl: (22) The requested URL returned error: 404 Not Found
Unknown ruby string (do not know how to handle): jruby-9.1.15.0.
Unknown ruby string (do not know how to handle): jruby-9.1.15.0.`
